include/ssl_key_handler.hpp

```diff
@@ -494,6 +494,13 @@ inline std::optional<boost::asio::ssl::context> getSSLClientContext()
         return std::nullopt;
     }
 
+    // Currently remote server's certificate verfication fails for
+    // HMC provided self-signed certificates,
+    // so skip remote server's certificate verfication.
+
+#if 0 // TODO: certificate verfication can be enabled after supporting
```

Comment:
did you try  mSslContext->add_verify_path(trustStorePath); to check if the verification works before disabling this?

Reply:
yes @abhilashraju  I have tried, there is no difference